CNJR-10789: Fix password and instructions to access the UI

kkonstantinov-cyberark opened the pull request:
password for admin was changed but not updated inside the set-up instructions

missing specific info on how to start the cluster, so we can login

kkonstantinov-cyberark committed:
Fix password and instructions to access the UI

jtuttle commented on demos/cluster/README.md:
```diff
@@ -89,7 +89,22 @@ $ bin/start --master-key --custom-certs --load-data
 
 ### Logging into your Cluster
 Navigate to [localhost](https://localhost) and login with the username: `admin`
```
Let's make this more succinct and mention an alternative for logging in for when --load-data isn't used. Something like:

You can now login to your cluster by either using a browser to navigate to [`localhost`](https://localhost) (only if you used the `--load-data` parameter) or by using the [Conjur CLI](https://docs.cyberark.com/conjur-open-source/latest/en/content/developer/cli/cli-setup.htm).

The username is ... and the password is ...

kkonstantinov-cyberark replied:
I think it won't be necessary stating about Conjur CLI, because the way bin/cli is set up it always logins before executing a given command(but it had issues with the flags which I fixed). 

Otherwise about the --load-data, I've tried to reproduce the issue, but apparently it may have been an issue on my side, but still the provided password in the README was incorrect and needed to be changed.

kkonstantinov-cyberark committed:
- Reverted to old text, but keep the new password as it is needed

- -i flag is incompatible with --self-signed, when using 'bin/cli' user won't be able to authenticate and cli will generate errors:
Error: Cannot specify both --insecure and --self-signed
Error: Unable to authenticate with Conjur. Please check your credentials.

using 'bin/cli conjur login' will not work

using correct flag now

jtuttle commented on demos/cluster/bin/cli:
```diff
@@ -19,7 +19,7 @@ function proxy_command {
   docker compose run --rm -w /src/cli --entrypoint /bin/sh client -c "
     if [ ! -e /root/conjur-server.pem ]; then
-      echo y | conjur init -i https://conjur-master-1.mycompany.local -a demo --self-signed
+      echo y | conjur init -u https://conjur-master-1.mycompany.local -a demo --self-signed
```
Was this not working before? Sure seems like `-i` is the wrong parameter.

kkonstantinov-cyberark replied:
Most probably it didn't work from POV of the CLI

<img width="798" height="168" alt="image" src="https://github.com/user-attachments/assets/06a77b44-df6c-4cbd-bbc3-6577fb9f1aba" />
I reverted the flag locally to provide screenshots of the CLI out.
User cannot even login through CLI 
<img width="806" height="202" alt="image" src="https://github.com/user-attachments/assets/f52eee2b-329d-4ce5-b4b1-b47d8ce78207" />


Now after changing the flag back to the correct one -u:
<img width="1000" height="440" alt="image" src="https://github.com/user-attachments/assets/9f8c53b4-7ffc-4bb2-a61d-79ce5c89adbb" />

and even though bin/cli automatically logs in we can still check just in case
<img width="997" height="354" alt="image" src="https://github.com/user-attachments/assets/f728b489-3fb9-4318-b14e-c89691519805" />


Otherwise on the UI side when using the new password everything is working.
